CONVERT_SERVICES/src/model/converterocr.py

```diff
@@ -0,0 +1,35 @@
+#
+# @converter_ocr.py Copyright (c)
+# 2643 Av  Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+#1376 Av General Inofuentes esquina calle 20, La Paz, Bolivia.
+#All rights reserved.
+#
+#This software is the confidential and proprietary information of
+#Jalasoft, ("Condidential Information"). You shall # not
+#disclose such Confidential Information and shall use it only in
+#accordance with the terms of the license agreement you entered into
+#with Jalasoft.
+#
+
+import pytesseract as tesseract_converter
+from PIL import Image
+from src.model.convertor import Convertor
+
+
+class ConvertOCR(Convertor):
```

Comment:
two names have to use a "_" to separate

Reply:
I can't change the name of class because in the past sprints defined this way for naming classes